pkg/api/addons_test.go

```diff
@@ -7249,25 +7246,7 @@ func TestSetAddonsConfig(t *testing.T) {
 			expectedAddons: []KubernetesAddon{
 				{
 					Name:    common.HeapsterAddonName,
-					Enabled: to.BoolPtr(true),
```

Comment:
This change is due to increasing the version to 1.13 (which makes sense because we no longer support 1.12)